content/preferences/index.md

```diff
@@ -98,9 +98,11 @@ These affect open and save operations.
 
 ### Recent Files History
 
-These change how the list of recent files is displayed in the File menu
+These change how the list of recent files ("MRU") is displayed in the File menu
```

Comment:
`MRU` is a common term to expert users; maybe not so much to noobs or intermediates.  If you are going to use it here, should it be defined here, or is it defined somewhere else?

Not really related, but I know that `MRU` is also used on the `MISC.` tab in the `Document Switcher` area...and here it is used in the UI of Notepad++ without definition!

Reply:
my intention had been for "list of recent files" to be the implied definition of "MRU"... hmm, maybe I should go make the misc section more explicit, and then link "MRU" to there, as well...

Reply:
I had intended for "list of recent files" to be the implicit definition for "MRU"... 
Ah, maybe I'll phrase it `list of recent files (also known as most-recently-used list, or "MRU" for short)`

Reply:
rephrased and committed.